tests/mock_s3_server/mock_s3_server.py

```diff
@@ -351,10 +399,18 @@ async def handle_mock_s3_request(wrapper, request):
         if type(event) is h11.EndOfMessage:
             break
         assert type(event) is h11.Data
+    if response_config is None:
```

Comment:
utterly trivial: space between blocks
```suggestion

    if response_config is None:
```